### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (p:nth-child(2))

### ISSUE THREAD

**Issue by animania4ka** [ada-cat/accessibility#6]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Ishinryu Karate SF](https://www.ishinryukaratesf.com/)
#### Element path
p:nth-child(2)
#### Snippet
`<p style="white-space:pre-wrap;">Saturday 11:30am</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 74.0.3729.108
This accessibility issue was found using Accessibility Insights for Web 2.2.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
